updates/cesium/20216-10-16-alejandro-garcia-salas.md

```diff
@@ -0,0 +1,8 @@
+#What did I do?
+Updated the readme with some more detailed instructions.
+
+#What I am planning to do?
+Dive into the source code and hopefully tackle a more interesting issue.
+
+#Issues:
+couldn't find the license for cesium-web for adding it to the readme.
```

Comment:
Please go ahead and copy the license file from Cesium: https://github.com/cesium-ml/cesium/blob/master/LICENSE.txt